xcodeproject/xcodeproj/target.go

```diff
@@ -43,18 +43,29 @@ func (t Target) DependentTargets() []Target {
 	return targets
 }
 
+// DependesOn ...
+func (t Target) DependesOn(targetID string) bool {
```

Comment:
Do we care about indirect dependencies (2 level down?)

Reply:
For now we do not.
iOS Auto Prov step only signs the UITest targets depending on the main application target.